doc/flume_extensions_catalogue/orion_sth_sink.md

```diff
@@ -17,7 +17,18 @@ Content:
     * [`MongoBackend` class](#section3.2)
 
 ##<a name="section1"></a>Functionality
-`com.iot.telefonica.cygnus.sinks.OrionSTHSink`, or simply `OrionSTHSink` is a sink designed to persist NGSI-like context data events within a MongoDB server in an aggregated way. Usually, such a context data is notified by a [Orion Context Broker](https://github.com/telefonicaid/fiware-orion) instance, but could be any other system speaking the <i>NGSI language</i>.
+`com.iot.telefonica.cygnus.sinks.OrionSTHSink`, or simply `OrionSTHSink` is a sink designed to persist NGSI-like context data events within a MongoDB server in an aggregated way, specifically these measures are computed:
+
+* For numeric attribute values:
+    * Sum of all the samples.
+    * Sum of the square value of all the samples.
+    * Maximum value among all the samples.
+    * Minimum value among all the samples.
+* Number of occurences for string attribute values.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/2d262bc5171759041c2b93dda3cc088c299d32d2